services/ui-src/src/components/cards/EntityCard/EntityCard.tsx

```diff
@@ -114,15 +114,22 @@ export const EntityCard = ({
           printVersion={!!printVersion}
         />
         {openAddEditEntityModal && (
-          <Button
-            variant="outline"
-            size="sm"
-            sx={sx.editButton}
-            leftIcon={<Image src={editIcon} alt="edit icon" height="1rem" />}
-            onClick={() => openAddEditEntityModal(entity)}
-          >
-            {verbiage.editEntityButtonText}
-          </Button>
+          <>
+            {!formattedEntityData.reportingPeriod ? (
+              <Text sx={sx.unfinishedMeasureMessage}>
+                {verbiage.unfinishedMeasureMessage}
+              </Text>
+            ) : null}
```

Comment:
Nit: Could use logical AND here instead of a ternary that returns `null`.

```suggestion
            {!formattedEntityData.reportingPeriod && (
              <Text sx={sx.unfinishedMeasureMessage}>
                {verbiage.unfinishedMeasureMessage}
              </Text>
            )}
```